packages/bot/src/components/contract-card.jsx

```diff
@@ -45,7 +42,7 @@ const ContractCard = ({
                 { is_contract_completed && <ContractResultOverlay profit={contract.profit} /> }
                 <div className='db-contract-card__underlying'>
                     <div className='db-contract-card__underlying-name'>
-                        <UnderlyingIcon market={contract.underlying} />
+                        <Icon icon={`IcUnderlying${contract.underlying}`} size={32} />
```

Comment:
If `contract.underlying` is `undefined` for whatever reason in `IcUnderlying${contract.underlying}` should it default to `IcUnknown`?

Reply:
sure, we can add an else statement in here to handle `undefined`